lib/ansible/modules/cloud/azure/azure_rm_virtualmachinescaleset.py

```diff
@@ -277,6 +280,7 @@
     vm_size: Standard_DS1_v2
     capacity: 2
     virtual_network_name: testvnet
+    upgrade_policy: Manual
```

Comment:
how the test passed before?

Reply:
I checked the integration test. All tests are provided this parameter `upgrade_policy`.